src/style.css

```diff
@@ -0,0 +1,94 @@
+@import url('https://fonts.googleapis.com/css2?family=Della+Respira&display=swap');
+
+:root {
+  --primary-color: #717754;
+}
+
+* {
+  font-family: 'Della Respira', serif;
+  box-sizing: border-box;
+}
+
+html {
+  color-scheme: dark;
+}
+
+h1 {
+  font-size: 55px;
+}
+
+section {
+  display: flex;
+  flex-direction: column-reverse;
+  justify-content: center;
+  align-items: center;
+}
+
+.container {
+  margin: 25px 0;
+  font-size: 1em;
+  min-width: 300px;
+  padding: 25px;
+  box-shadow: 7px 7px 21px #333, -7px -7px 21px #222;
+  border-radius: 20px;
+  overflow: hidden;
+}
+
+.styled-table {
+  border-collapse: collapse;
+}
+
+.styled-table thead tr {
+  background-color: var(--primary-color);
+  color: #fff;
+}
+
+.styled-table th,
+.styled-table td {
+  padding: 15px 25px;
+}
+
+.styled-table tbody tr {
+  border-bottom: 1px solid #ddd;
+}
+
+.styled-table tbody tr:nth-of-type(even) {
+  /* background-color: #f3f3f3; */
```

Comment:
Kindly take away commented blocks or pieces of codes from your codebase as they make it messy and give it a less professional look😉.